### PR TITLE
Adds List View Styles and Content

### PR DESCRIPTION
**Pull request by Lorraine-Lyu**
Pre-set list length 
polish styles and adds more fields to colleges.

image

### CODE REVIEW

**Review by MikolJiron — APPROVED** [review]
This looks awesome! We might need to coordinate how our changes will be merged together.

File: src/main/webapp/js/datahandlers/collegelistdatahandler.js [hunk 1]
```diff
@@ -59,8 +59,10 @@ class CollegeListDataHandler extends ListDataHandler {
     return [
       element[ID].toString(),
       element[NAME],
-      element[ACCEPTANCE_RATE].toString(),
-      element[ACT_SCORE].toString(),
+      `Accepatance Rate: ${(element[ACCEPTANCE_RATE]*100)
```
Comment (CarlosM10011) [inline src/main/webapp/js/datahandlers/collegelistdatahandler.js]: typo.

**Review by CarlosM10011 — CHANGES_REQUESTED** [review]
Some linter messages:

```
src/main/webapp/js/datahandlers/collegelistdatahandler.js:22: WARNING - [JSC_REQUIRES_NOT_SORTED] goog.require() and goog.requireType() statements are not sorted. The correct order is:

const {ACCEPTANCE_RATE, ACT_SCORE, CITY, CollegeQueryBuilder, ID, NAME, STATE} = goog.require('datahandlers.collegequerybuilder');
const {ListDataHandler} = goog.require('datahandlers.listdatahandler');

  22| const {ACCEPTANCE_RATE, ACT_SCORE, CITY, STATE, CollegeQueryBuilder, ID, NAME} = goog.require('datahandlers.collegequerybuilder');
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/main/webapp/js/datahandlers/collegequerybuilder.js:27: WARNING - [JSC_MISSING_SEMICOLON] Missing semicolon
  27| const CITY = 'school.city'
      ^^^^^^^^^^^^^^^^^^^^^^^^^^

src/main/webapp/js/datahandlers/scholarshiplistdatahandler.js:30: WARNING - [JSC_UNUSED_LOCAL_ASSIGNMENT] Value assigned to local variable SCHOOLS_LIST is never read
  30| const SCHOOLS_LIST = 'schoolsList';
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: src/main/webapp/js/finscholar/commonlistview/commonlistview.js [hunk 2]
```diff
@@ -162,6 +162,12 @@ class CommonListView extends BasicView {
     }
   }
 
+  setHeight_() {
```
Comment (CarlosM10011) [inline src/main/webapp/js/finscholar/commonlistview/commonlistview.js:210]: add jsdoc plese

**Review by CarlosM10011 — CHANGES_REQUESTED** [review]
More linter messages:

```
src/main/webapp/js/datahandlers/scholarshiplistdatahandler.js:29: WARNING - [JSC_UNUSED_LOCAL_ASSIGNMENT] Value assigned to local variable SCHOOL_NAME is never read
  29| const SCHOOL_NAME = 'school names';
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: src/main/webapp/js/finscholar/commonlistview/commonlistview.js [hunk 3]
```diff
@@ -162,6 +162,9 @@ class CommonListView extends BasicView {
     }
   }
 
+  /**
+   * @private Preset the hight of the list.
```
Comment (CarlosM10011) [inline src/main/webapp/js/finscholar/commonlistview/commonlistview.js]: Have the docstring above the access specifier

```
  /**
   * Preset the hight of the list.
   * @private
   */
```

**Review by CarlosM10011 — CHANGES_REQUESTED** [review]

**Review by CarlosM10011 — APPROVED** [review]

File: src/main/webapp/js/datahandlers/collegelistdatahandler.js [hunk 4]
```diff
@@ -59,8 +59,10 @@ class CollegeListDataHandler extends ListDataHandler {
     return [
       element[ID].toString(),
       element[NAME],
-      element[ACT_SCORE].toString(),
-      element[ACCEPTANCE_RATE].toString()
+      `Acceptance Rate: ${(element[ACCEPTANCE_RATE]*100)
+                                .toFixed(/** precision= */ 2)}%`,
```
Comment (tinagrasso) [inline src/main/webapp/js/datahandlers/collegelistdatahandler.js]: create a util.

File: src/main/webapp/js/datahandlers/collegequerybuilder.js [hunk 5]
```diff
@@ -38,19 +39,19 @@ const ACCEPTANCE_RATE = 'latest.admissions.admission_rate.overall';
...
+const FIRST_NET_COST = '2016.cost.net_price.private.by_income_level.0-30000';
+const SECOND_NET_COST =
+    '2016.cost.net_price.private.by_income_level.30001-48000';
+const THIRD_NET_COST =
+    '2016.cost.net_price.private.by_income_level.48001-75000';
+const FOURTH_NET_COST =
+    '2016.cost.net_price.private.by_income_level.75001-110000';
+const FIFTH_NET_COST =
+    '2016.cost.net_price.private.by_income_level.110001-plus';
+const MEDIAN_DEBT = '2016.aid.median_debt.completers.overall';
```
Comment (tinagrasso) [inline src/main/webapp/js/datahandlers/collegequerybuilder.js]: why is this being changed to old data?

File: src/main/webapp/js/finscholar/commonlistview/commonlistview.soy [hunk 6]
```diff
@@ -33,22 +33,28 @@
...
-  <p> Loading more items ... </p>
+  <div id="loading">
+    <div id="p2" class="mdl-progress mdl-js-progress mdl-progress__indeterminate"></div>
+  </div>
 {/template}
 
 {template .endoflist}
-  <p> This is the end of the list. </p>
+  <div id="list-inner-footer">
+    <p> This is the end of the list. </p>
```
Comment (tinagrasso) [inline src/main/webapp/js/finscholar/commonlistview/commonlistview.soy]: nit: can you change the wording to something more intuitive like "Displaying all items." Optionally, I don't think you need a footer at all.

**Review by tinagrasso — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/finscholar/commonlistview/commonlistview.soy (hunk 6 above)
Comment (Lorraine-Lyu) [reply src/main/webapp/js/finscholar/commonlistview/commonlistview.soy]: Sure fixed.

File: src/main/webapp/js/datahandlers/collegequerybuilder.js (hunk 5 above)
Comment (Lorraine-Lyu) [reply src/main/webapp/js/datahandlers/collegequerybuilder.js]: Oh I didn't realized the change when rebasing to devel.
Just fixed it

File: src/main/webapp/js/finscholar/commonlistview/commonlistview.soy (hunk 6 above)
Comment (tinagrasso) [reply src/main/webapp/js/finscholar/commonlistview/commonlistview.soy]: not seeing any change.

**Review by tinagrasso — CHANGES_REQUESTED** [review]

**Review by tinagrasso — APPROVED** [review]

**Review by hzimz — APPROVED** [review]

**Pull request merged**